### PR TITLE
Access from URL

### PR DESCRIPTION
**Pull request by HarryBosworth**
- Merged Okoyes changes and the template.
- Added start page.
- Added config validation.

Resolves: [AOAF-220](https://companieshouse.atlassian.net/browse/AOAF-220)

[AOAF-220]: https://companieshouse.atlassian.net/browse/AOAF-220?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .vscode/launch.json
Comment (okoye-companieshouse) [inline .vscode/launch.json:1]: Do we need to merge this file?

File: .eslintrc.old
Comment (okoye-companieshouse) [inline .eslintrc.old:1]: Should this be deleted?

Comment (HarryBosworth) [reply .eslintrc.old:1]: Yes I'll make that change.

File: Tiltfile.dev
[the remark this answers is not in the document]
Comment (HarryBosworth) [reply Tiltfile.dev:1]: No I'll revert that

**Review by okoye-companieshouse — APPROVED** [review]

**Pull request merged**